Paintroid/src/main/java/org/catrobat/paintroid/model/Layer.kt

```diff
@@ -19,25 +19,16 @@
 package org.catrobat.paintroid.model
 
 import android.graphics.Bitmap
-import android.graphics.Color
 import org.catrobat.paintroid.contract.LayerContracts
 
-open class Layer(override var bitmap: Bitmap?) : LayerContracts.Layer {
-    override var transparentBitmap: Bitmap? = null
+open class Layer(override var bitmap: Bitmap) : LayerContracts.Layer {
```

Comment:
the `bitmap` can be nullable, for eg. check `OpenRasterFileFormatConversion` line 276

Reply:
for this reason `importOraFile` throws a nullpointerexception.

Reply:
hmm ok, there's a null check implemented there, missed that.